CustomScript/customscript.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
This script needs to call through the shim.  Python3 is not available on all distros, and the intent is to make customscript.py portable to Python2 only or Python3 only distros.

I don't think this change is necessary at all because shim.sh is executing customscript.py directly.

Reply:
This was a mistake (I was testing something else).
Removed!